lib/sensu-plugins-cassandra/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsCassandra
   module Version
     MAJOR = 2
     MINOR = 1
-    PATCH = 0
```

Comment:
Can you please remove this change, again we will update version numbers when we roll up a release.

Reply:
This has been done